command_sign_test.go

```diff
@@ -37,8 +46,8 @@ func TestSignIncludeCertsAIA(t *testing.T) {
 	require.NoError(t, err)
 
 	require.Equal(t, 2, len(certs))
```

Comment:
Note that we already check the chain size, so we'll still have a test that the chains are "equal" to what is expected.